api/environments/models.py

```diff
@@ -88,6 +90,21 @@ def create_feature_states(self):
                 else feature.default_enabled,
             )
 
+    @property
+    def last_updated_at(self) -> typing.Optional[datetime]:
+        # NOTE: `last_updated_at` is a proxy for changes that affect a value of a given flag in the environment
+        last_audit_log = (
+            AuditLog.objects.exclude(
```

Comment:
I makes me nervous doing database queries in `property`s - I'd rather this was a regular method.

Reply:
In fact, especially since the AuditLog table is pretty hefty, I'd rather avoid doing queries against that table where we can avoid it. Couldn't we make this a concrete field on the Environment model and use a `post_save` hook on the AuditLog model to update it?

Reply:
Yeah, good point